app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -151,6 +162,10 @@ class WireActivityViewModel @Inject constructor(
         navigateTo(NavigationCommand(NavigationItem.IncomingCall.getRouteWithArgs(listOf(conversationId))))
     }
 
+    private fun openConversation(conversationId: ConversationId) {
+        navigateTo(NavigationCommand(NavigationItem.Conversation.getRouteWithArgs(listOf(conversationId)), BackStackMode.UPDATE_EXISTED))
```

Comment:
maybe for later: we can extract all this model creation to the navigation class or somewhere and here only pass the conversation id.